app/controllers/projects/extracts_controller.rb

```diff
@@ -29,12 +29,11 @@ def authenticate
 
   def find_info_request
     if params[:url_title]
-      @info_request = @project.info_requests.find_by!(
+      @info_request = @project.info_requests.extractable.find_by!(
         url_title: params[:url_title]
       )
     else
-      # HACK: Temporarily just find a random request to render
-      @info_request = @project.info_requests.sample
+      @info_request = @project.info_requests.extractable.first
```

Comment:
Should continue to use `sample` here because we don't want people landing on the same request all the time.

Reply:
Oops that was a mistake